rcpchgrowth/rcpchgrowth/unit_tests/test_sds_calculation.py

```diff
@@ -4,49 +4,75 @@
 from datetime import datetime
 from ..date_calculations import chronological_decimal_age, corrected_decimal_age, estimated_date_delivery
 
+
 class TestSDS(unittest.TestCase):
 
-    ## Tests for term children
+    # Tests for length/height:
+
+    def test_sds_term_boy_at_30weeks_length(self):
+
+      # calculate the length sds of a boy born at 30 weeks
+
+        z = sds(-0.095824778, 'height', 46.1415, 'male')
+        self.assertEqual(z, 0)
```

Comment:
after discussion with @statist7 the tests need to be refactored slightly, to accept a value as correct when it is withing a 'bracket' or range of values, for which we think an accuracy of +/- 10^-5 is acceptable,